### ISSUE TITLE
Please allow to not show minutes and seconds

### ISSUE THREAD

**Issue by firebreez3** [pmongloid/flipdown-timer-card#29]
**Is your feature request related to a problem? Please describe.**
I want to show a countdown of seconds only but would love the flexibility to allow options for minutes and seconds.

**Describe the solution you'd like**
show_minute   |  string   |  Optional   |  Show minute rotors true, false, auto
show_second   |  string   |  Optional   |  Show second rotors true, false, auto

**Describe alternatives you've considered**
I removed hours but to fit in a block the height cuts off the top of the text

**Additional context**
screenshot attached
![flip timer 30 seconds](https://user-images.githubusercontent.com/98600531/214061339-ff690a8a-667c-4e5f-b98e-daacb2999b3f.png)

**Comment by loryanstrant** [comment]
Agree, would love to be able to hide the seconds rotors in some instances.

**Comment by radimsejk** [comment]
I would like this option too. It would be great to show seconds only when remaining itme is less than a minute.

**Comment by ThomasHeinrichSchmidt** [comment]
Same here. Is it possible to have `show_minutes `and `show_seconds`? Would be great.
